dbt_project.yml

```diff
@@ -801,6 +801,9 @@ models:
       polygon:
         +schema: rocifi_polygon
 
+    impossible_finance:
+      +schema: impossible_finance
```

Comment:
please add subdirectory for bnb here, as that is where model lives & add schema for `impossible_finance_bnb`